elastic/kibana
# [Fleet] Allow +build versions when upgrading agents

## Summary

Closes https://github.com/elastic/kibana/issues/189752

Allow agent upgrades to newer `+build` versions of agent.

This required bumping the `semver` package and its types to make use of the new `includePrerelease` flag.

## To test

1. Enroll an agent on `8.14.3`
2. Select the "upgrade agent" action in the UI
3. Enter `8.14.3+build202407291657` as the new version
4. Observe the agent upgrades successfully

## Checklist

Delete any items that are not applicable to this PR.
